app/components/pages/SubmissionWizard/index.js

```diff
@@ -88,6 +88,7 @@ const SubmissionWizard = ({ match, history }) => (
           )}
         />
         <Route
+          path={`${match.path}/author`}
           render={() => (
             <WizardStep
               component={AuthorPageContainer}
```

Comment:
Would be good to also add a [redirect](https://reacttraining.com/react-router/web/api/Redirect) from `/submit/:id` to `/submit/:id/author`.